src/content_handlers/apt_handler/src/apt_parser.cpp

```diff
@@ -88,6 +92,18 @@ std::unique_ptr<AptContent> GetAptContentFromRootValue(JSON_Value* rootValue)
                     ADUC_ERC_APT_HANDLER_INVALID_PACKAGE_DATA);
             }
 
+            // Note: For backward compatibility.
+            //
```

Comment:
what's the backwards compatibility case?

Reply:
The idea is to check package's name for "deviceupdate-agent". Incase customer doesn't explicitly specify "agentRestartRequired" in the manifest. 
(As you mentioned in other comment).

I can remove this, if you think that this doesn't provide any value. 

---
In reply to: [596183648](https://github.com/Azure/iot-hub-device-update/pull/17#discussion_r596183648) [](ancestors = 596183648)